test/e2e/bundle_image_test.go

```diff
@@ -276,7 +276,7 @@ func pushLoadImages(client *kubernetes.Clientset, w io.Writer, images ...string)
 		}
 	} else {
 		for _, image := range images {
-			pushWith("docker", image)
+			err = pushWith("docker", image)
```

Comment:
nit:

```
if err := pushWith("docker", image); err != nil {
	return err
}
```

Reply:
err is declared in line 262.  Is there some benefit to re-defining it in this scope? (I'd missed this)